public/css/style.css

```diff
@@ -0,0 +1,177 @@
+@font-face {
```

Comment:
Why do we have both a `main.css` and `style.css`? Is one file meant for a certain type of styling? If so, we should mention that and rename the files to be more specific so future contributors know how to organize the CSS